src/main/java/com/bakdata/kafka/KafkaApplication.java

```diff
@@ -80,8 +74,8 @@ public abstract class KafkaApplication implements Runnable {
                     + "consumer group. Be careful with running in production and with enabling this flag - it "
                     + "might cause inconsistent processing with multiple replicas.")
     protected boolean cleanUp;
-    @CommandLine.Option(names = "--schema-registry-url", required = true, description = "URL of schema registry")
-    private String schemaRegistryUrl = "";
+    @CommandLine.Option(names = "--schema-registry-url", description = "URL of schema registry")
+    private Optional<String> schemaRegistryUrl;
```

Comment:
Using optionals for fields is bad practice

Reply:
But this is supported in [PicoCLI](https://picocli.info/#_optionalt). Should I consider a default `null` value instead?

Reply:
Just a plain String field. Will be null by default anyways. Just because PicoCLI supports it, it is still bad practice because Optional is not Serializable for example. IntelliJ will warn you as well